series/APILibrary.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import json
+from .show import Show
+from datetime import datetime
+
+
+api_url = "https://api.themoviedb.org/3/"
+api_key = "?api_key=b6f2e2170c53099a5a65cf3d336d242f"
+language = "language=en-US"
+
+icon_url = "https://image.tmdb.org/t/p/"
+small_size = "w154"
+big_size = "w300/"
+
+
+def search_show(title: str):
+    """
+    search title in API
+
+    :param title: string
+    :return: array of Show
+    """
+    endpoint = "search/tv"
+    url = api_url + endpoint + api_key + "&" + language + "&query=" + title + "&page=1"
```

Comment:
Does `page=1` means we only get the first page of search results, i.e. there may be more we don't get?

I suppose that's fine for now but worth documenting (kind comment) if we want to support loading multiple pages of search results in the future.